main.go

```diff
@@ -315,7 +317,21 @@ func main() {
 			log.Printf("%v\n", err)
 			os.Exit(64)
 		}
-		exitCode := RunWorker()
+		monitor = NewMonitor(
+			config.Project,
+			tcauth.New(&tcclient.Credentials{
+				ClientID:    config.ClientID,
+				AccessToken: config.AccessToken,
+				Certificate: config.Certificate,
+			}),
+			config.LogLevel,
+			nil,
+		)
+
+		exitCode := INTERNAL_ERROR
+		incidentID := monitor.CapturePanic(func() {
+			exitCode = RunWorker()
+		})
```

Comment:
Tests don't expect RunWorker to panic, since RunWorker doesn't panic. However, they check the exit code that RunWorker returns, and if a panic occurs, this will be captured in the exitCode. They do check this. So really it is about the RunWorker method being designed to return a code based on what happened (idle timeout, worker requests a reboot, worker crashed, job was successful, etc) - and tests check that. Does that make sense?